static/src/javascripts/projects/common/utils/inlineSvg.js

```diff
@@ -20,7 +20,7 @@ define([
                 dirs = path.split("/"),
                 fileName = dirs.pop();
 
-            text.get(req.toUrl(dirs.join("/") + "/" + imageType + "/" + fileName + ".svg"), function (svg) {
+            text.get(req.toUrl(dirs.join("/") + "/" + (imageType ? imageType + "/" : "") + fileName + ".svg"), function (svg) {
```

Comment:
The tests were failing without this change. Does the plugin still need this `imageType` magic?